packages/spell-check/0.1.0/README.md

```diff
@@ -0,0 +1,3 @@
+This is a very powerful typo-correction library. It contains hundreds of thousands of typo fixes. The library comes from another developer, so credit where it is due: https://github.com/platers/obsidian-linter/blob/master/src/utils/auto-correct-misspellings.ts
+
+There are also a few custom ones at the bottom. If you are experiencing issues with typing a word and that has two capital letters at the beginning being replaced with lowercase, remove the very last addition.
```

Comment:
Maybe highlight the mention of `# Fix Double Capitals` specifically for Windows users, for whom it won't work.